openedx/core/lib/xblock_pipeline/finder.py

```diff
@@ -116,7 +116,7 @@ def __init__(self, *args, **kwargs):
         """
         The XBlockPipelineFinder creates a separate XBlockPackageStorage for
         every installed XBlock package when its initialized. After that
-        initialization happens, we just proxy all list()/find() requests by
+        initialization happens, we just proxy all []/find() requests by
```

Comment:
This seems like a global search/replace that was over-broad.

Reply:
Reverted the change.